cpp/src/fil/infer.cu

```diff
@@ -238,19 +238,27 @@ struct tree_aggregator_t {
     : tmp_storage(finalize_workspace) {}
 
   __device__ __forceinline__ void accumulate(
-    vec<NITEMS, float> single_tree_prediction, int tree, int num_rows) {
```

Comment:
Why is `num_rows` no longer needed?

Reply:
it wasn't needed before either, but in this PR, I needed to either remove it or give a value for how many rows the thread? block? should process. Since it would need to acquire a new meaning in this PR (rows/thread vs rows/block), giving it any value would be confusing. Hence, just deleted it. I do not see it being needed with any changes on our to-do list.